README.md

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
Download: Should we use the 3.x specific download link instead?

Docs: It won't be possible to point to the latest version for the docs ^ use 3.0.0-beta1.